web-marketplace/src/hooks/useDaoOrganization.ts

```diff
@@ -0,0 +1,16 @@
+import { useAuth } from 'lib/auth/auth';
+import { useMemo } from 'react';
+
+export const useDaoOrganization = () => {
```

Comment:
this hook provides easy access to the current organization, which can be useful for APP-789